Included response output: spo externaluser until spo hidedefaultthemes commands Closes #3930 (#5542)
pnp/cli-microsoft365: docs/docs/cmd/spo/file/file-checkin.mdx

```md definition-list
`-u, --webUrl <webUrl>`
: The URL of the site where the file is located.

`--url [url]`
: The server- or site-relative decoded URL of the file to retrieve. Specify either `url` or `id` but not both.

`-i, --id [id]`
: The UniqueId (GUID) of the file to retrieve. Specify either `url` or `id` but not both.

`-t, --type [type]`

> : Type of the check in. Available values `Minor`, `Major`, `Overwrite`. Default is `Major`.

Reviewer: ```suggestion
: Type of the check in. Allowed values: `Minor`, `Major`, `Overwrite`. Defaults to `Major`.
```
